precompile/contracts/rewardmanager/contract.go

```diff
@@ -25,7 +25,11 @@ const (
 	AreFeeRecipientsAllowedGasCost uint64 = allowlist.ReadAllowListGasCost
 	CurrentRewardAddressGasCost    uint64 = allowlist.ReadAllowListGasCost
 	DisableRewardsGasCost          uint64 = (contract.WriteGasCostPerSlot) + allowlist.ReadAllowListGasCost // write 1 slot + read allow list
-	SetRewardAddressGasCost        uint64 = (contract.WriteGasCostPerSlot) + allowlist.ReadAllowListGasCost // write 1 slot + read allow list
+
+	DuplicatedLogGas      uint64 = 375 // TODO: duplicated from params/protocol_params.go to avoid import cycle in tests.
+	DuplicatedLogTopicGas uint64 = 375 // TODO: duplicated from params/protocol_params.go to avoid import cycle in tests.
```

Comment:
@ceyonur temporary code duplication to avoid import cycle in tests. Not sure how to solve this.

Reply:
We can move `LogGas` and `LogTopicGas` to a new package. Both packages that need the constants can import?

Reply:
I think its worth pulling out `precompile_config_test.go` out of params and into a new package. It also might be worth moving the specific precompile tests out of `config.test.go`. Best to keep params/protocol_params.go as synced to geth. @ceyonur knows best though